Dockerfile

```
@@ -0,0 +1,9 @@
+FROM golang:1.16
+
+WORKDIR /app
+
+COPY . .
+
+RUN go build -o /walrus-server cmd/server/v1/main.go
+
+ENTRYPOINT ["/walrus-server"]
```

I would like if the go version is still 1.17 as before, you may need to run `go mod tidy`. Could you also change the binary name to just `server`? The same in the Makefile's compile command
```suggestion
FROM golang:1.17

WORKDIR /app

COPY . .

RUN go build -o /server cmd/server/v1/main.go

ENTRYPOINT ["/server"]
```

---

I commited the change.

However I still think it is wrong to set the Go version in `go.mod` at 1.17 when it does not work immediately if you try to compile the program. The developer should not have to fix the dependencies in order to make the program to compile. It should work by default. I think we need an issue about this.

---

Dude, it is just run a go mod tidy on your machine